.github/workflows/test-kind.yaml

```diff
@@ -20,11 +20,12 @@ jobs:
           k8sVersion: ${{ matrix.k8sVersion }}
     - name: Kind Cluster
```

Comment:
Should we still call this `test-kind`? We can probably update to just call it the name that we intend to use long-term

Reply:
Good call. I've renamed it to `kind-e2e`